src/dropdown/dropdownbuttonview.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+import ButtonView from '/ckeditor5/ui/button/buttonview.js';
+
+/**
+ * The basic dropdown button view class.
+ *
+ * @memberOf ui.dropdown
+ * @extends ui.button.ButtonView
+ */
+
```

Comment:
We rather avoid leaving this blank line between the API doc string and the thing being documented. There are some inconsistencies now, but in >90% there are no blank lines.